src/sets.jl

```diff
@@ -235,12 +235,23 @@ dual_set(s::RotatedSecondOrderCone) = copy(s)
 """
     GeometricMeanCone(dimension)
 
-The geometric mean cone ``\\{ (t,x) \\in \\mathbb{R}^{n+1} : x \\ge 0, t \\le \\sqrt[n]{x_1 x_2 \\cdots x_n} \\}`` of dimension `dimension```{}=n+1``.
+The geometric mean cone ``\\{ (t,x) \\in \\mathbb{R}^{1+n} : x \\ge 0, t \\le \\sqrt[n]{x_1 x_2 \\cdots x_n} \\}`` of dimension `dimension```{}=1+n``.
 """
 struct GeometricMeanCone <: AbstractVectorSet
     dimension::Int
 end
 
+"""
+    GeneralizedGeometricMeanCone{T <: Real}(exponents::Vector{T})
```

Comment:
How would we name the power cone where we have ||x|| in the rhs ?

Reply:
According to https://en.wikipedia.org/wiki/Generalized_mean#Geometric_mean, we should call it WeightedPowerMeanCone